src/include/execution_kernel.hpp

```diff
@@ -456,6 +462,10 @@ __global__ void executionKernel([[maybe_unused]] int rank /*for debug*/, T* inpu
       T* dst = getBuffer(input, output, scratch, op.dstBufferType);
       T* src = getBuffer(input, output, scratch, op.srcBufferType);
       handleCopyPacket<PacketType>(dst, src, scratchSize, op.dstOffset, op.srcOffset, op.size, flag);
+    } else if (op.type == OperationType::COPY_TO_PACKET) {
+      T* dst = getBuffer(input, output, scratch, op.dstBufferType);
+      T* src = getBuffer(input, output, scratch, op.srcBufferType);
+      handleCopyPacket<PacketType>(dst, src, scratchSize, op.dstOffset, op.srcOffset, op.size, flag);
```

Comment:
`handleCopyToPacket` ? Maybe we can give a better name for this op, such as TransformToPacket?

Reply:
Sure, I'll update this and create a new PR. Thank you!